2023/03/19 16:58:38 [error] 17#17: *10 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /var/www/app/vendor/yiisoft/yii2/base/Controller.php:149
Stack trace:
#0 /var/www/app/vendor/yiisoft/yii2/base/Module.php(552): yii\base\Controller->runAction('error', Array)
#1 /var/www/app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction('site/error')
#2 /var/www/app/vendor/yiisoft/yii2/base/ErrorHandler.php(152): yii\web\ErrorHandler->renderException(Object(yii\web\NotFoundHttpException))
#3 [internal function]: yii\base\ErrorHandler->handleException(Object(yii\web\NotFoundHttpException))
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "requests/". in /var/www/app/vendor/yiisoft/yii2/base/Module.php:561
Stack trace:
#0 /var/www/app/vendor/yiisoft/yii2/web/Application.php(103): yii\base\Module->runAction('requests/', Array)
#1 /var/www/app/vendor/yiisoft/yii2/base/Application.php(384): yii\we" while reading upstream, client: 172.18.0.1, server: , request: "GET /requests/ HTTP/1.1", upstream: "fastcgi://unix:/var/run/php-fpm.sock:", host: "localhost:8080"
2023/03/19 16:58:51 [error] 17#17: *14 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /var/www/app/vendor/yiisoft/yii2/base/Controller.php:149
Stack trace:
#0 /var/www/app/vendor/yiisoft/yii2/base/Module.php(552): yii\base\Controller->runAction('error', Array)
#1 /var/www/app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction('site/error')
#2 /var/www/app/vendor/yiisoft/yii2/base/ErrorHandler.php(152): yii\web\ErrorHandler->renderException(Object(yii\web\NotFoundHttpException))
#3 [internal function]: yii\base\ErrorHandler->handleException(Object(yii\web\NotFoundHttpException))
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "requests/". in /var/www/app/vendor/yiisoft/yii2/base/Module.php:561
Stack trace:
#0 /var/www/app/vendor/yiisoft/yii2/web/Application.php(103): yii\base\Module->runAction('requests/', Array)
#1 /var/www/app/vendor/yiisoft/yii2/base/Application.php(384): yii\we" while reading upstream, client: 172.18.0.1, server: , request: "GET /requests/ HTTP/1.1", upstream: "fastcgi://unix:/var/run/php-fpm.sock:", host: "localhost:8080"
2023/05/03 13:30:40 [error] 17#17: *8 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /var/www/app/vendor/yiisoft/yii2/base/Controller.php:149
Stack trace:
#0 /var/www/app/vendor/yiisoft/yii2/base/Module.php(552): yii\base\Controller->runAction('error', Array)
#1 /var/www/app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction('site/error')
#2 /var/www/app/vendor/yiisoft/yii2/base/ErrorHandler.php(152): yii\web\ErrorHandler->renderException(Object(yii\web\NotFoundHttpException))
#3 [internal function]: yii\base\ErrorHandler->handleException(Object(yii\web\NotFoundHttpException))
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "admin/". in /var/www/app/vendor/yiisoft/yii2/base/Module.php:561
Stack trace:
#0 /var/www/app/vendor/yiisoft/yii2/web/Application.php(103): yii\base\Module->runAction('admin/', Array)
#1 /var/www/app/vendor/yiisoft/yii2/base/Application.php(384): yii\web\Appl" while reading upstream, client: 172.18.0.1, server: , request: "GET /admin/ HTTP/1.1", upstream: "fastcgi://unix:/var/run/php-fpm.sock:", host: "localhost:8080"
2023/05/03 13:31:28 [error] 17#17: *10 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /var/www/app/vendor/yiisoft/yii2/base/Controller.php:149
Stack trace:
#0 /var/www/app/vendor/yiisoft/yii2/base/Module.php(552): yii\base\Controller->runAction('error', Array)
#1 /var/www/app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction('site/error')
#2 /var/www/app/vendor/yiisoft/yii2/base/ErrorHandler.php(152): yii\web\ErrorHandler->renderException(Object(yii\web\NotFoundHttpException))
#3 [internal function]: yii\base\ErrorHandler->handleException(Object(yii\web\NotFoundHttpException))
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "admin/index". in /var/www/app/vendor/yiisoft/yii2/base/Module.php:561
Stack trace:
#0 /var/www/app/vendor/yiisoft/yii2/web/Application.php(103): yii\base\Module->runAction('admin/index', Array)
#1 /var/www/app/vendor/yiisoft/yii2/base/Application.php(384): yi" while reading upstream, client: 172.18.0.1, server: , request: "GET /admin/index HTTP/1.1", upstream: "fastcgi://unix:/var/run/php-fpm.sock:", host: "localhost:8080"
2023/05/03 13:31:51 [error] 17#17: *12 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /var/www/app/vendor/yiisoft/yii2/base/Controller.php:149
Stack trace:
#0 /var/www/app/vendor/yiisoft/yii2/base/Module.php(552): yii\base\Controller->runAction('error', Array)
#1 /var/www/app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction('site/error')
#2 /var/www/app/vendor/yiisoft/yii2/base/ErrorHandler.php(152): yii\web\ErrorHandler->renderException(Object(yii\web\NotFoundHttpException))
#3 [internal function]: yii\base\ErrorHandler->handleException(Object(yii\web\NotFoundHttpException))
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "admin/". in /var/www/app/vendor/yiisoft/yii2/base/Module.php:561
Stack trace:
#0 /var/www/app/vendor/yiisoft/yii2/web/Application.php(103): yii\base\Module->runAction('admin/', Array)
#1 /var/www/app/vendor/yiisoft/yii2/base/Application.php(384): yii\web\Appl" while reading upstream, client: 172.18.0.1, server: , request: "GET /admin/ HTTP/1.1", upstream: "fastcgi://unix:/var/run/php-fpm.sock:", host: "localhost:8080"
2023/05/03 13:32:20 [error] 17#17: *17 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /var/www/app/vendor/yiisoft/yii2/base/Controller.php:149
Stack trace:
#0 /var/www/app/vendor/yiisoft/yii2/base/Module.php(552): yii\base\Controller->runAction('error', Array)
#1 /var/www/app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction('site/error')
#2 /var/www/app/vendor/yiisoft/yii2/base/ErrorHandler.php(152): yii\web\ErrorHandler->renderException(Object(yii\web\NotFoundHttpException))
#3 [internal function]: yii\base\ErrorHandler->handleException(Object(yii\web\NotFoundHttpException))
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "admin". in /var/www/app/vendor/yiisoft/yii2/base/Module.php:561
Stack trace:
#0 /var/www/app/vendor/yiisoft/yii2/web/Application.php(103): yii\base\Module->runAction('admin', Array)
#1 /var/www/app/vendor/yiisoft/yii2/base/Application.php(384): yii\web\Applic" while reading upstream, client: 172.18.0.1, server: , request: "GET /admin HTTP/1.1", upstream: "fastcgi://unix:/var/run/php-fpm.sock:", host: "localhost:8080"
